doc/asciidoc/docker/index.adoc

```diff
@@ -288,4 +288,67 @@ Inside the directory `/kafka-connect-neo4j/docker` you'll find a compose file th
 include::../../../kafka-connect-neo4j/docker/docker-compose.yml[]
 ----
 
-include::../../../kafka-connect-neo4j/docker/readme.adoc[]
\ No newline at end of file
+include::../../../kafka-connect-neo4j/docker/readme.adoc[]
+
+=== Neo4j Streams with Neo4j Cluster and Kafka Cluster
+
+Here we provide a docker-compose file to quickstart with an environment composed by a 3-nodes Neo4j Causal Cluster
+(with Streams plugin configured in Sink mode) and a 3-nodes Kafka Cluster.
+
+[source,yaml]
+----
+include::data/docker-compose-cluster.yml[]
+----
+
+What you need to do is just:
+
+* Download the latest Neo4j Streams plugin version from here: https://github.com/neo4j-contrib/neo4j-streams/releases/tag/4.0.0
```

Comment:
It's 4.0.1 now.  ;)

Reply:
I think that there's a little problem, because in the "Releases" section the downloadable JAR, associated to the latest plugin version, is still the 4.0.0